js/components/operatorSignIn/attestation.tsx

```diff
@@ -139,17 +161,19 @@ const SignatureHint = ({
   );
 };
 
-export const SignaturePrompt = ({
+export const InputBox = ({
   onChange,
   defaultValue,
+  title,
 }: {
-  onChange: (value: string) => void;
+  onChange: (evt: ChangeEvent<HTMLInputElement>) => void;
```

Comment:
optional refactoring suggestion:
`InputBox` isn't used outside of this file, so doesn't need the `export`.
`onChange` might be cleaner if it passed the `evt.target.value` as a `string`, instead of passing the whole event and making the caller get the value. That way the event details stay closer to the HTML input that generated them.